tests/RecurlyResource_Test.php

```diff
@@ -44,6 +44,22 @@ public function testFromJsonValidResource(): void
         $this->assertInstanceOf(\Recurly\Resources\TestResource::class, $result->getSingleChild());
     }
 
+    public function testCast(): void
+    // accepts response object of decoded json
+    // returns RecurlyResource
+    {
+        $test_response_object = (object)array(
+            "id" => 0,
+            "object" => "plan",
+            "name" => "0",
+        );
+
+        $resource = Recurly\Resources\TestResource::cast($test_response_object);
+        $this->assertNotNull($resource->getName());
```

Comment:
I would assert that the getter returns the expected value instead of asserting that it's not null. In theory, `getName()` could return the integer `0` and the test would pass.